packages/devtools_app/lib/src/network/network_request_inspector_views.dart

```diff
@@ -151,6 +160,86 @@ class HttpResponseView extends StatelessWidget {
   }
 }
 
+class ImageResponseView extends StatelessWidget {
+  const ImageResponseView(this.data);
+
+  final DartIOHttpRequestData data;
+
+  @override
+  Widget build(BuildContext context) {
+    final img = image.decodeImage(data.encodedResponse);
+    return Column(
+      crossAxisAlignment: CrossAxisAlignment.stretch,
+      children: [
+        _buildTile(
+          'Image Preview',
+          [
+            Padding(
+              padding: const EdgeInsets.all(
+                denseSpacing,
+              ),
+              child: Image.memory(
+                data.encodedResponse,
+              ),
+            ),
+          ],
+        ),
+        LayoutBuilder(builder: (context, constraints) {
+          return _buildTile(
+            'Metadata',
+            [
+              _buildRow(
+                context,
+                'Format',
+                data.type,
+              ),
+              _buildRow(
+                context,
+                'Size',
+                prettyPrintBytes(
+                  data.encodedResponse.lengthInBytes,
+                  includeUnit: true,
+                ),
+              ),
+              _buildRow(
+                context,
+                'Dimensions',
+                '${img.width} x ${img.height}',
+              ),
+            ],
+          );
+        }),
+      ],
+    );
+  }
+
+  Widget _buildRow(
+    BuildContext context,
+    String key,
+    String value,
+  ) {
+    return Container(
```

Comment:
change to Padding widget since padding is the only property being used from Container